tests/Task.test.ts

```diff
@@ -603,4 +631,41 @@ describe('toggle done', () => {
             todaySpy.mockClear();
         },
     );
+
+    it('supports recurrence rule after a due date', () => {
+        // Arrange
+        const line =
+            '- [ ] this is a task 🗓 2021-09-12 ✅ 2021-06-20 🔁 every day';
+        const path = 'this/is a path/to a/file.md';
+        const sectionStart = 1337;
+        const sectionIndex = 1209;
+        const precedingHeader = 'Eloquent Section';
+
+        // Act
+        const task = Task.fromLine({
+            line,
+            path,
+            sectionStart,
+            sectionIndex,
+            precedingHeader,
+        });
```

Comment:
As per the earlier comment, it would be helpful for this to be simplified by using the test helper.

Reply:
`s/earlier/later/` - the comments have appeared in time order, not line order... Sorry for the confusion.

Reply:
> As per the earlier comment, it would be helpful for this to be simplified by using the test helper.

This one slipped through the net (the order of comments was confusing) - but don't worry, I will tidy up a whole load of instances of complicated Task creation in this file, to increase the chances of future tests also being simplified, so I will fix it then.